packages/camera/camera_android_camerax/example/android/build.gradle

```diff
@@ -29,3 +29,15 @@ subprojects {
 task clean(type: Delete) {
     delete rootProject.buildDir
 }
+
+// Build the plugin project with warnings enabled. This is here rather than
+// in the plugin itself to avoid breaking clients that have different
+// warnings (e.g., deprecation warnings from a newer SDK than this project
+// builds with).
+gradle.projectsEvaluated {
+    project(":camera_android_camerax") {
+        tasks.withType(JavaCompile) {
+            options.compilerArgs << "-Xlint:all" << "-Werror"
```

Comment:
Are the compiler args we want in this repo always these 2? If so is there a way to share the configuration? 

It is ok to merge without resolving this.

Reply:
We don't currently have any way to share configuration between different packages; I don't know enough about gradle to know if there's a reasonable way to do that (e.g., if we can reference random files outside of the build).

In general, we haven't invested much in managing the example projects on any platform. `flutter_migrate` was something we'd discussed as a future option for some of that kind of thing, but we could also make repo tooling to mass-modify our examples if it's useful.

Reply:
ok if we dont already have a system then this pr is not the place to build one.